promptsource/templates.py

```diff
@@ -67,7 +69,7 @@ def remove_template(self, dataset, template_name):
             raise ValueError(f"No template with name {template_name} " + f"for dataset {dataset} exists.")
 
         del self.templates[dataset][template_name]
-
+        self.templates[dataset]["count"] = num_templates
```

Comment:
Is it necessary to maintain an explicit count of the templates? Would just calling `len(self.templates[dataset])` have the same effect and avoid the headache of having to keep track of the counts?